<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditor</class>
 <widget class="QMainWindow" name="MapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="map"/>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="aType">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Player</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Orc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Troll</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HealthPotion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LightingBoltScroll</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FireballScroll</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ConfusionScroll</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Door</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenDoor</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Actor</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="posX">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="posY">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuMap"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
